scripts/deploy/messageBridge.ts

```diff
@@ -25,17 +25,15 @@ export async function deployMessageBridge(managementAddress: string, deployer: W
     console.log("Message Bridge Logic deployed at: ", await upgrades.erc1967.getImplementationAddress(await msgBridge.getAddress()));
 
     console.log("\n# Message Bridge Configuration");
-    const managementSlot = "0xd6595d2280e6cba67baf67ff997445e733b244161e59228efeb7032069381100";
-    const management = await ethers.provider.getStorage(await msgBridge.getAddress(), managementSlot);
-    console.log("Management Slot:                  ", management);
-    const msgBridgeState = await msgBridge.getMessageBridgeState();
-    console.log("Message Bridge Paused:            ", msgBridgeState.paused);
-    console.log("Message Bridge Sending Paused:    ", msgBridgeState.sendingPaused);
-    console.log("Message Bridge Executing Paused:  ", msgBridgeState.executingPaused);
-    console.log("Message Bridge Sending Fee:       ", msgBridgeState.config.fee);
-    console.log("Message Bridge Max Msg Size:      ", msgBridgeState.config.maxMessageSize);
-    console.log("Message Bridge Max Nr Messages:   ", msgBridgeState.config.maxNrMessages);
-    console.log("Message Bridge Execution Window:  ", msgBridgeState.config.executionWindowSeconds);
+    console.log("Management Slot:                  ", await msgBridge.management());
+    console.log("Message Executor Slot:            ", await msgBridge.executionManager());
```

Comment:
The execution manager has not been set yet at this point. Move this to the last line of the `deployMessageBridgeContracts()` function right after it has been set. Also, rename:

```suggestion
    console.log("ExecutionManager:                   ", await msgBridge.executionManager());
```